src/qt/addressbookpage.cpp

```diff
@@ -328,3 +319,16 @@ void AddressBookPage::selectNewAddress(const QModelIndex &parent, int begin, int
         newAddressToSelect.clear();
     }
 }
+
+void AddressBookPage::updateWindowsTitleWithWalletName()
+{
+    const QString walletName = this->model->GetWalletDisplayName();
+
+    if (mode == ForEditing) {
+        switch(tab)
+        {
+        case SendingTab: setWindowTitle(tr("Sending addresses - %s").arg(walletName)); break;
+        case ReceivingTab: setWindowTitle(tr("Receiving addresses - %s").arg(walletName)); break;
```

Comment:
```suggestion
        case SendingTab: setWindowTitle(tr("Sending addresses - %1").arg(walletName)); break;
        case ReceivingTab: setWindowTitle(tr("Receiving addresses - %1").arg(walletName)); break;
```

Please refer to https://doc.qt.io/qt-5/qstring.html#arg

Reply:
This works now.